rust/fastsim-core/src/simdrive/simdrive_impl.rs

```diff
@@ -632,20 +632,11 @@ impl RustSimDrive {
             }
         }
         // Is SOC below min threshold?
-        if self.soc[i - 1] < (self.veh.min_soc + self.veh.perc_high_acc_buf) {
-            self.reached_buff[i] = false;
-        } else {
-            self.reached_buff[i] = true;
-        }
+        self.reached_buff[i] = self.soc[i - 1] >= (self.veh.min_soc + self.veh.perc_high_acc_buf);
```

Comment:
Sick!!

Reply:
@kylecarow , all thanks to `cargo clippy`